setup.py

```diff
@@ -205,7 +214,7 @@ class ONNXExtension(setuptools.Extension):
     def pre_run(self):
         pass
 
-def create_extension(ExtType, name, sources, dependencies, extra_link_args, extra_objects):
+def create_extension(ExtType, name, sources, dependencies, extra_link_args, extra_objects, onnx_ml=False):
```

Comment:
It should be `create_extension` taking `define_macros` as argument and the caller of `create_extension` compute the macros and passing it in.

Reply:
fixed. Thanks!